framework/src/main/java/framework/db/SonarDbClient.java

```diff
@@ -189,7 +189,7 @@ private IComponent parseComponentFromQuery(ResultSet queryResult) throws SQLExce
 					.setPackageName(packageName)
 					.setSuperClass(superclass)
 					.setInterfaces(Lists.newArrayList(Splitter.on(",").split(interfaces)))
-					.setIsInterface(isInterface)
+					.setIsInterface(isInterface > 0)
```

Comment:
Až tu budou konstanty, tak bude lepší použít  isInterface == IS_INTERFACE (možná to chce lepší název konstanty)